### PR TITLE
feat: add nibiru blockchain to chain registry

### PR DESCRIPTION
**Pull request by k-yang**
Adds the Nibiru blockchain to the Cosmos Chain Registry

### CODE REVIEW

File: nibiru/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      ],
+      "base": "unibi",
+      "name": "Nibiru",
+      "display": "nibi",
+      "symbol": "NIBI",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/nibiru.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/nibiru.svg"
+      },
+      "coingecko_id": "nibiru",
```
Comment (JeremyParish69) [inline nibiru/assetlist.json:25]: This coingecko id doesn't exist. Please remove

File: nibiru/chain.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nibiru",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://nibiru.fi/",
+  "pretty_name": "Nibiru",
+  "chain_id": "cataclysm-1",
+  "bech32_prefix": "nibi",
```
Comment (JeremyParish69) [inline nibiru/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (k-yang) [reply nibiru/chain.json:9]: Pending PR: https://github.com/satoshilabs/slips/pull/1647

File: nibiru/assetlist.json (hunk 1 above)
Comment (k-yang) [reply nibiru/assetlist.json]: Fixed: https://www.coingecko.com/en/coins/nibi

**Comment by Unique-Divine** [comment]
- Closes https://github.com/NibiruChain/nibiru/issues/536

**Pull request merged**